### DISCUSSION TITLE
GitHub Pro account does not allow me to access 'Network' in 'Insights'

### DISCUSSION THREAD

**Discussion by FD81** [community/community#66964] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have purchased GitHub Pro so that I can access the "network" feature of "Insights" for my private repositories, yet this is not available.
Does anyone know why?

This is a screenshot FYI:

![image](https://github.com/community/community/assets/86591054/a762d1ec-9dfa-4860-99cc-bb6964059c91)

**Comment by vedantswain** [comment]
I am encountering the same issue. 
Were you able to resolve this issue?
